sympy/solvers/tests/test_solvers.py

```diff
@@ -1298,8 +1298,16 @@ def test(e1, e2):
 
 def test_check_assumptions():
     x = symbols('x', positive=True)
-    assert solve(x**2 - 1) == [1]
```

Comment:
AFAIK, `solve` internally uses `check_assumptions` to check whether a solution is valid. If this isn't tested in another place, I think we should leave it here.

Reply:
yeah thanks! I missed this. I include it.